OnboardPietroPieri/components/user-list.tsx

```diff
@@ -10,40 +9,71 @@ import {
 import { User } from "../interfaces/mutation";
 import { USERS } from "../apollo/query";
 import { useQuery } from "@apollo/client";
-import { Users } from "../interfaces/query";
-import AsyncStorage from "@react-native-async-storage/async-storage";
+import { PageInput, Users } from "../interfaces/query";
+import { pageInputIni } from "../utils/pages";
 
-export const UserListScreen = () => {
-  const [token, setToken] = useState<string | null>(null);
+interface UserListProps {
+  token: string | null;
+}
 
-  useEffect(() => {
-    const getToken = async () => {
-      const userToken = await AsyncStorage.getItem("userToken");
-      setToken(userToken);
-    };
-
-    getToken();
-  }, []);
+export const UserList: React.FC<UserListProps> = ({ token }) => {
+  const [loadingMore, setLoadingMore] = useState(false);
+  const [initialLoading, setInitialLoading] = useState(true);
+  const page = useRef<PageInput>(pageInputIni);
```

Comment:
Evitamos usar abreviações, no geral. Só teria que escrever um pouco a mais e facilita muito a leitura
```suggestion
  const page = useRef<PageInput>(pageInputInitialValue);
```

Reply:
Aplicado